packages/@lwc/aria-reflection-polyfill/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "@lwc/aria-reflection-polyfill",
+    "version": "2.22.0",
+    "description": "ARIA element reflection polyfill for strings",
+    "homepage": "https://lwc.dev/",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/salesforce/lwc.git",
+        "directory": "packages/@lwc/aria-reflection-polyfill"
+    },
+    "bugs": {
+        "url": "https://github.com/salesforce/lwc/issues"
+    },
+    "main": "dist/index.cjs",
+    "module": "dist/index.js",
+    "license": "MIT",
+    "typings": "types/index.d.ts",
+    "scripts": {
+        "clean": "rm -rf dist/ types/",
+        "build": "rollup --config scripts/rollup.config.js",
+        "dev": "rollup  --config scripts/rollup.config.js --watch --no-watch.clearScreen"
+    },
+    "files": [
+        "dist/"
+    ],
+    "keywords": [
+        "lwc",
+        "aria",
+        "polyfill",
+        "reflection",
+        "aom"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "dependencies": {
+        "@lwc/shared": "2.22.0"
+    }
```

Comment:
In the `dist` files, `@lwc/shared` is imported, which makes it a dependency, not a devDependency. The reason we do this is so that `@lwc/shared` is not duplicated in `engine-dom.js`.